Makefile

```diff
@@ -165,8 +165,8 @@ minikube-install: gadget-container kubectl-gadget
 	# because of Finalizers.
 	kubectl delete crd traces.gadget.kinvolk.io || true
 	./kubectl-gadget deploy | kubectl delete -f - || true
-	time kubectl wait --for=delete namespace gadget 2>/dev/null || true
-	time kubectl wait --for=delete daemonset -n kube-system gadget 2>/dev/null || true
+	time kubectl wait --for=delete --timeout=60s namespace gadget 2>/dev/null || true
+	time kubectl wait --for=delete daemonset -n gadget gadget 2>/dev/null || true
```

Comment:
Is it necessary to delete the gadget DaemonSet now that it is in the gadget namespace? The previous line should be deleting it already.